traveler-ios-ui/TravelerKitUI/ModelExtensions/BookingItemDetails+TravelerUI.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 import TravelerKit
 
-extension CatalogItemDetails {
+extension BookingItemDetails {
```

Comment:
I actually think ALL CatalogItemDetails should have this

Isn't `description` part of `CatalogItemDetails`?